File: app/src/main/java/com/walkercoding/videocompress/videocompress/LogUtils.java
package com.walkercoding.videocompress.videocompress;

import android.util.Log;

public class LogUtils {
    public static void e(final Throwable e) {
        Log.e("zl", "Error throwable: " + e.getMessage());
    }

    public static void i(final String message) {
        Log.e("zl", "Error msg: " + message);
    }
}

File: app/src/main/java/com/walkercoding/videocompress/videocompress/LogUtils.kt
package com.walkercoding.videocompress.videocompress

import android.util.Log

object LogUtils {
    fun e(e: Throwable) {
        Log.e("zl", "Error throwable: " + e.message)
    }

    fun i(message: String) {
        Log.e("zl", "Error msg: $message")
    }
}
